src/pages/settings/Profile/Contacts/ContactMethodDetailsPage.js

```diff
@@ -183,6 +224,14 @@ class ContactMethodDetailsPage extends Component {
         const formErrorText = this.state.formError ? this.props.translate(this.state.formError) : '';
         const isFailedAddContactMethod = Boolean(lodashGet(loginData, 'errorFields.addedLogin'));
 
+        // Users are only allowed to change their default contact method to the current one if:
+        // 1. This contact method is not already their default
+        // 2. This contact method is validated
+        // 3. Default contact method switching is allowed by their domain security group (if this exists)
+        const canChangeDefaultContactMethod = !isDefaultContactMethod
```

Comment:
Why aren't `!isDefaultContactMethod` and `loginData.validatedDate` just not conditions included inside `getCanChangeDefaultContactMethod()`? Instead of a new const, why not just rename the method to `canChangeDefaultContactMethod()` and call it directly?

Reply:
Iiiiii think that would be clearer, good call both of y'all :D